CodeStarterCamp_Week4/startRoutine.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  routineCount.swift
+//  CodeStarterCamp_Week4
+//
+//  Created by JSB on 2023/03/15.
+//
+
+import Foundation
+
+struct Routine {
+    let routineName: String
+    var activities: [Activity]
+    
+    func startAndThrows(_ inputString: String?, _ bodyCondition: BodyCondition, _ routine: Routine) throws {
+        
+        do {
+            let input = try checkInputValue(inputString)
+            print("\(routine.routineName)을 시작합니다.")
+            
+            for routineCount in 1...input {
+                print("\(routineCount)번째 루틴을 시작합니다.")
+                print("-------------------------")
+                
+                for activity in routine.activities {
+                    activity.action(bodyCondition)
+                    print("-------------------------")
+                    guard bodyCondition.피로도 < 100 else {
+                        throw RoutineInputError.fatigueOver100
+                    }
+                }
+            
+                bodyCondition.checkBodyCondition()
+                print("-------------------------")
+            }
+        }
+    }
+
+    func start(bodyCondition: BodyCondition, routine: Routine) {
+        
+        while(true) {
```

Comment:
```suggestion
        while true {
```

이러한 단일 조건에 대해서는 보통 소괄호를 사용하지 않고, 사이에 공백이 들어갑니다 😲